amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/oas/parser/document/Oas3DocumentParser.scala

```diff
@@ -41,7 +50,7 @@ case class Oas3DocumentParser(root: Root)(implicit override val ctx: OasWebApiCo
       parseExamplesDeclaration(map, parent + "/examples")
       parseLinkDeclarations(map, parent + "/links")
       super.parseSecuritySchemeDeclarations(map, parent + "/securitySchemes")
-      super.parseTypeDeclarations(map, parent + "/types", Some(this))
+      super.parseTypeDeclarations(map, Some(this))
```

Comment:
This could break Visual Designer logic. VS is creating apis using the object oriented model, and trying to replicate our way to generate ids.

Reply:
id adoption inside amf parser code is useless as we apply final id adoption after.